Backpack-SwiftUI/Skeleton/Classes/BPKSkeleton.swift

```diff
@@ -0,0 +1,131 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import Backpack_Common
+
+public struct BPKSkeleton: View {
+    private var type: BPKSkeleton.ViewType = .image
+    private var size: BPKSkeleton.Size = .none
```

Comment:
This could be simplified, adding a `customSize(CGSize)` case on the Size enum. we won't need an extra `customSize` property here plus the additional init won't have to exist!

logic down this struct will be simplified also

Reply:
Good catch, this is more Swifty! You are really an iOS expert!

Reply:
☺️